src/opnsense/scripts/unbound/blocklists.py

```diff
@@ -147,7 +147,6 @@ def uri_reader(uri):
     # write out results
     with open("/usr/local/etc/unbound.opnsense.d/dnsbl.conf", 'w') as unbound_outf:
         if blocklist_items:
-            unbound_outf.write('server:\n')
```

Comment:
what's the story here?

Reply:
server: is a no-op. It does nothing, the original thought was that a `server:` statement was necessary to parse the local-data entries. This makes parsing easier.

Reply:
thanks for clarification :)